src/notation/internal/masternotation.cpp

```diff
@@ -77,8 +78,8 @@ void MasterNotation::setMasterScore(Ms::MasterScore* score)
     }
 
     setScore(score);
-
     initExcerptNotations(masterScore()->excerpts());
+    m_notationMidiData->init(m_parts);
```

Comment:
I think, better to pass m_parts to MasterNotationMidiData as a parameter in constructor like m_notationChanged
and there init what you need by parts changed notification

Then you do not need to call notationMidiData::init in several places

Reply:
might make sense, thanks

Reply:
Actually, no

I have to be sure that the pointer on score is valid at the moment of initialization. I could move parts subscriptions to the constructor, but it doesn't make sense to process them if I have no score at the moment.

I'll leave it as it was